### ISSUE TITLE
Correct Language codes in Gutenberg recipes

### ISSUE THREAD

**Issue by RavanJAltaie** [openzim/gutenberg#217]
For Gutenberg, we use the "one-language-one-zim" mode in Zimfarm. In this mode, the language is set automatically by the scraper. Obviously the scraper is creating ZIMs with improper language => open upstream issue in Gutenberg scraper, nothing you can solve yourself.

there are two issues:

- `openZIM:gutenberg_mul_all` is improper ZIM name, `mul` language is not a valid ISO-639-3 language code
- `openZIM:gutenberg_rmr_all` is improper ZIM name, `rmr` language is not a valid ISO-639-3 language code anymore ; as of 2010-01-18, [rmr] for Caló is deprecated due to split. split into Caló [rmq] and Erromintxela [emx]

**Comment by eshellman** [comment]
I can see about Caló (it's only one book) from upstream, but none of the others are language codes from PG, that I know of.

**Comment by benoit74** [comment]
Thank you @eshellman, if you could fix `rmr` upstream it would be great ; otherwise we would have to add a "hack" to our scraper to transform `rmr` into `rmq,emx` since it's probably the real situation, or maybe only `rmq`

`mul` is a hack for the ZIM we create with all languages. The scraper should not do that to respect openZIM specification, and list all languages. This part is for us ^^

**Comment by rgaudin** [comment]
@benoit74 Languages metadata must be a list of ISO-639-3 **sorted by importance** (so number of entries here) but the Name metadata and the filename will keep the mul.

**Comment by benoit74** [comment]
> Languages metadata must be a list of ISO-639-3 sorted by importance (so number of entries here) but the Name metadata and the filename will keep the mul.

Yep, I had this in mind. Thank you for confirming before I even asked 😄

**Comment by benoit74** [comment]
(and sorry for the wrong description in first comment, I wrote it too fast)

**Comment by benoit74** [comment]
Some part of this issue has been fixed upstream: https://aleph.pglaf.org/cache/epub/29470/pg29470.rdf is now referencing language as `rmq`.

We still need to fix the Language metadata of the `mul` ZIM.

**Comment by benoit74** [comment]
`Language` metadata of `mul` ZIM is already OK:

https://github.com/openzim/gutenberg/blob/6049ced0cb7548e54d863e67ce02a6e59d584271/src/gutenberg2zim/zim.py#L36-L49

**Comment by benoit74** [comment]
Moved everything left to zim-requests since there is no scraper issue in fact.
